solr/src/main/conf/schema.xml

```diff
@@ -0,0 +1,613 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<!--
+ Licensed to the Apache Software Foundation (ASF) under one or more

```

Comment:
It has been copied but changed (a lot) to support the default field names that the ycsb client uses i.e. "field0" - "field9". It is being included to help illustrate the importance and connection between Solr's schema.xml file and ycsb's expected field names as an example only. This was added in response to a previous comment regarding the clarity and important of the schema.xml file in Step 2 of the README.

I might replace the file with an original simplified snippet that likely won't need reference to any license if that makes things simpler?